engines/e_capi.c

```diff
@@ -472,10 +476,9 @@ static int capi_init(ENGINE *e)
     }
 # endif
 
-    /* See if we support AES CSP */
-
-    if (CryptAcquireContext(&hprov, NULL, NULL, PROV_RSA_AES,
-                            CRYPT_VERIFYCONTEXT)) {
+    /* See if there is RSA+AES CSP */
+    if (CryptAcquireContextW(&hprov, NULL, rsa_aes_cspname, PROV_RSA_AES,
```

Comment:
From https://msdn.microsoft.com/en-us/library/windows/desktop/aa379886(v=vs.85).aspx

"If this parameter is NULL, the user default provider is used."